requirements/requirements.txt

```diff
@@ -2,4 +2,5 @@
 neon-mq-connector>=0.7.2a1
 neon_utils[sentry]==1.11.1a5
 ovos-config~=0.0.10
-pydantic==2.6.3
\ No newline at end of file
+pydantic~=2.6
+neon-data-models@git+https://github.com/neongeckocom/neon-data-models@FEAT_LLMRequestModels
```

Comment:
lets first merge this PR and then publish it to PyPi to further get the actual version there

Reply:
PyPI upload will fail with git dependencies, so the `neon-data-models` PR must be merged before this one.